src/modules/assetPack/sagas.ts

```diff
@@ -71,8 +71,9 @@ export function* assetPackSaga(builder: BuilderAPI) {
       yield put(setProgress(ProgressStage.UPLOAD_CONTENTS, 0))
 
       const updatableAssets = assetPack.assets.filter(asset => Object.keys(contents[asset.id]).length > 0)
-      const onProgress = yield handleAssetContentsUploadProgress(updatableAssets.length)
-      const uploadEffects = updatableAssets.map(asset => builder.saveAssetContents(asset, contents[asset.id]).then(onProgress))
+      const onProgress: void = yield handleAssetContentsUploadProgress(updatableAssets.length)
+      // TODO: there's a bug here
+      const uploadEffects = updatableAssets.map(asset => builder.saveAssetContents(asset, contents[asset.id]).then(onProgress as any))
```

Comment:
This refers to the `onProgress as any` but I didn't have time to investigate it.